src/Network/Probe.php

```diff
@@ -333,7 +333,7 @@ public static function uri($uri, $network = "", $uid = 0, $cache = true) {
 			/// The biggest problem is the avatar picture that could have a reduced image size.
 			/// It should only be updated if the existing picture isn't existing anymore.
 			if (($data['network'] != NETWORK_FEED)
-				AND ($mode == PROBE_NORMAL)
+				AND ($data["network"] != NETWORK_DIASPORA)
```

Comment:
Because the `$mode` parameter doesn't exist in `Probe::uri()`, so he replaced it with the behavior from `probe_url()`